content/switching-to-v0-mangling-on-nightly.md

```diff
@@ -0,0 +1,171 @@
++++
+path = "2025/10/28/switching-to-v0-mangling-on-nightly"
+title = "Switching to Rust's own mangling scheme on nightly"
+authors = ["David Wood"]
+
+[extra]
+team = "the compiler team"
+team_url = "https://www.rust-lang.org/governance/teams/compiler"
++++
+
+**TL;DR:** rustc will use its own "v0" mangling scheme by default on nightly
```

Comment:
```suggestion
**TL;DR:** rustc will use its own "v0" symbol mangling scheme by default on nightly instead of the legacy C++-compatible one
```

Reply:
I don't think we should say "compatible" because someone might read it as this makes us incompatible with C++. Maybe just "instead of the previous default scheme which reused C++ mangling"?

Reply:
Added